src/Generator.php

```diff
@@ -139,7 +139,11 @@ protected function populateServers()
     {
         if (config('l5-swagger.paths.base') !== null) {
             if ($this->isOpenApi()) {
-                $this->swagger->servers = [
+                if (! is_array($this->swagger->servers)) {
+                    $this->swagger->servers = [];
+                }
+
+                $this->swagger->servers[] = [
                     new \OpenApi\Annotations\Server(['url' => config('l5-swagger.paths.base')]),
                 ];
```

Comment:
I think it should be:

```php
$this->swagger->servers[] = new \OpenApi\Annotations\Server(['url' => config('l5-swagger.paths.base')]);
```

Reply:
In the beginning, I think it should be like you mentioned. But, I found if there was not `@OA\Server` annotation, the value of `$this->swagger->servers` is `@OA\UNDEFINED` string. If `$this->swagger->servers` was appended, it show the following error:

```
1) Tests\GeneratorTest::canGenerateApiJsonFileWithChangedBaseServer
Error: [] operator not supported for strings
```

Reply:
But you have this `if (! is_array($this->swagger->servers))` which should owerrite string to array?

Reply:
Yes. in the line 142, it changes string to array. If I misunderstand your concerns, please let me know.

Reply:
if we do like this (like in your PR):
```php
$this->swagger->servers[] = [new \OpenApi\Annotations\Server(['url' => config('l5-swagger.paths.base')])];
```
The result of `$this->swagger->servers` will be:
```php
[
   //Custom server1,
   //Custom server2,
   [
      new \OpenApi\Annotations\Server(['url' => config('l5-swagger.paths.base')])
   ]
]
```

But previously it was:
```php
[
   new \OpenApi\Annotations\Server(['url' => config('l5-swagger.paths.base')])
]
```

IMHO final array should look like:
```php
[
   //Custom server1,
   //Custom server2,
   new \OpenApi\Annotations\Server(['url' => config('l5-swagger.paths.base')])
]
```

Reply:
@DarkaOnLine Sorry, I had not found the `\OpenApi\Annotations\Server` object exists in an array. `$this->swagger-servers` should be: 

```
[
   //Custom server1,
   //Custom server2,
   new \OpenApi\Annotations\Server(['url' => config('l5-swagger.paths.base')])
]
```

And it's appropriate.

I think I was dizzy in afternoon. :persevere: